tests/unit/test_options.py

```diff
@@ -7,6 +8,23 @@
 from tests.lib.options_helpers import AddFakeCommandMixin
 
 
+@contextmanager
+def assert_raises_message(exc_class, expected):
```

Comment:
This is a nice helper; maybe in the future we should look into if this can be used elsewhere in the test suite.